pkg/generator.go

```diff
@@ -201,6 +201,9 @@ func (g *Generator) mockName() string {
 		if ast.IsExported(g.iface.Name) {
 			return "Mock" + g.iface.Name
 		}
+		if g.Exported {
+			return "Mock" + strings.Title(g.iface.Name)
```

Comment:
Actually after looking at this again, would it make more sense to instead just do this instead?

```go
if g.Exported or ast.IsExported(g.iface.Name) {
	return "Mock" + g.iface.Name
}
```
The casing of the interface name doesn't matter since we prepend with `Mock` which makes it exported. Less logic would be better. What is your opinion?

Reply:
Agree, updated the PR.

Reply:
Great, once PR builder is done I'll merge.